### PR TITLE
Add tailwind to mainLayOut and NavBar

### PR DESCRIPTION
**Pull request by kavinda-100**
## Purpose
The Navbar is not responsive when resizing the browser window.
(https://github.com/sef-global/scholarx-frontend/issues/73)

## Goals
add tailwindcss utility classes to MainLayOut and NavBar to fix the issue.

## Approach
 I had set the max width to the nav bar and added proper Tailwind classes according to the device breakpoints to achieve responsiveness for the NavBar.

## Screenshots
https://github.com/sef-global/scholarx-frontend/issues/73#issuecomment-1904086007

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kavinda-100** [comment]
add tailwind branch

File: src/components/Layout/MainLayout.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React, { type ReactNode } from 'react';
 import { Layout } from 'antd';
 
 import FooterComponent from './Footer/Footer';
-import styles from './MainLayout.module.css';
+// import styles from './MainLayout.module.css';
```
Comment (anjula-sack) [inline src/components/Layout/MainLayout.tsx]:
```suggestion
```

File: src/components/Layout/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -68,106 +68,112 @@ const Navbar: React.FC = () => {
...
-                <Text className={styles.antTypography}>About</Text>
+                <Text className="antTypography text-primary text-[14px] font-light">About</Text>
               </a>
               <a href="" target="_blank" rel="noreferrer">
-                <Text className={styles.antTypography}>Initiatives</Text>
+                <Text className="antTypography text-primary text-[14px] font-light">Initiatives</Text>
               </a>
               <a href="" target="_blank" rel="noreferrer">
-                <Text className={styles.antTypography}>Join Us</Text>
+                <Text className="antTypography text-primary text-[14px] font-light">Join Us</Text>
```
Comment (anjula-sack) [inline src/components/Layout/Navbar/Navbar.tsx]: Use a standard size like `text-sm` `text-md`

File: tailwind.config.js [hunk 3]
```diff
@@ -4,6 +4,8 @@ export const theme = {
   extend: {
     colors: {
       'primary-blue': '#1677FF',
+      'ant-icon-color': '#525F7F',
```
Comment (anjula-sack) [inline tailwind.config.js]: Lets use a different name here without the ant part

```suggestion
      'ant-icon-color': '#525F7F',
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Comment by anjula-sack** [comment]
@kavinda-100 There are some conflicts can you fix them?

**Comment by anjula-sack** [comment]
@kavinda-100 There are some conflicts can you fix them as well?

**Comment by kavinda-100** [comment]
done the required changes

**Comment by anjula-sack** [comment]
@kavinda-100 The conflicts are still there

**Review by anjula-sack — CHANGES_REQUESTED** [review]
@mayura-andrew There are some lint issues.

**Comment by anjula-sack** [comment]
Closing due to inactivity

**Pull request closed without merging**